blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -55,6 +60,19 @@ export class Pipelines extends Component {
         });
     }
 
+    handleKeyDownEvent = (event) => {
+        if (document.activeElement !== document.getElementsByClassName('fastsearch-input')[0]) {
```

Comment:
I'd add a function called "getSearchInput" which contains this code to avoid duplicating it in three places.